aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -167,6 +167,21 @@ Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& ma
   at::ScalarType scalar_type = self.scalar_type();
   c10::MaybeOwned<Tensor> self_;
   if (&result != &self) {
+#if defined(USE_ROCM) && ROCM_VERSION >= 50600
+    if (std::getenv("USE_HIPBLASLT")){
```

Comment:
I would check the value as well otherwise `USE_HIPBLASLT=0` also enables hipblasLt